content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -42,7 +42,8 @@ $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | cut -f1-1 -d "." | \
 Restore if necessary:
 
 {{< text bash >}}
-$ kubectl apply -f $HOME/ISTIO_1_0_RESTORE_CRD_DATA.yaml
+$ grep -v selfLink /home/sdake/ISTIO_1_0_RESTORE_CRD_DATA.yaml | grep -v resourceVersion | \
```

Comment:
```suggestion
$ grep -v selfLink $HOME/ISTIO_1_0_RESTORE_CRD_DATA.yaml | grep -v resourceVersion | \
```

Reply:
Actually, can't this just be one grep command?
```
$ grep -v "selfLink|resourceVersion|creationTimestamp|uid|generation" $HOME/ISTIO_1_0_RESTORE_CRD_DATA.yaml | kubectl apply -f -
```
```

Reply:
Also, are you sure that using `kubectl get --export=true` in the backup command doesn't solve this problem?

Reply:
@frankbu I tried that grep command as one, but it iddn't seem to work.  I will try again.

I also tried --export=true, didn't seem to do the job.  @howardjohn had same results.

Reply:
@frankbu with your grep command the CR on a restore looks like
```
apiVersion: v1
items:
- apiVersion: networking.istio.io/v1alpha3
  kind: DestinationRule
  metadata:
    annotations:
      kubectl.kubernetes.io/last-applied-configuration: |
        {"apiVersion":"networking.istio.io/v1alpha3","kind":"DestinationRule","metadata":{"annotations":{},"name":"outlier","namespace":"default"},"spec":{"host":"google.com","trafficPolicy":{"outlierDetection":{"baseEjectionTime":"15m","consecutiveErrors":7,"interval":"5m"}}}}
    creationTimestamp: "2019-03-19T04:04:14Z"
    generation: 1
    name: outlier
    namespace: default
    resourceVersion: "6623"
    selfLink: /apis/networking.istio.io/v1alpha3/namespaces/default/destinationrules/outlier
    uid: 0f501338-49fc-11e9-83fb-0042688109a5
```

Reply:
this however looks good ```grep -v "selfLink\|resourceVersion\|creationTimestamp\|uid\|generation" $HOME/ISTIO_1_0_RESTORE_CRD_DATA.yaml > zz
```